exercises/beer-song/src/test/java/BeerSongTest.java

```diff
@@ -9,39 +9,39 @@
     public void singFirstVerse() {
         assertEquals("99 bottles of beer on the wall, 99 bottles of beer.\n" +
                         "Take one down and pass it around, 98 bottles of beer on the wall.\n\n",
-                BeerSong.verse(99));
+                new BeerSong().verse(99));
```

Comment:
If you make a private BeerSong variable, then you can initialise it in a setup method and just do `beerSong.verse(99)` here :) That way you don't have to duplicate the initialisation in every test :)